base_app/src/app/app.component.ts

```diff
@@ -158,24 +158,27 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
 
     }
 
-    static getDataTabRelativePath(relativeURL, tab_index, tab_hash, serverSideJsDebugging) {
-
-        // Default assumes relativeURL is a complete URL
+    static getDataTabPath(incomingURL, tab_index, tab_hash, serverSideJsDebugging: boolean) {
+        // TODO: verify that the hash stuff isn't being used for incremental update logic
+        //       and remove.
```

Comment:
still todo?

Reply:
Yes - want to leave this comment for right now.  Will fix in the future.